pkgs/by-name/sv/svndumpsanitizer/package.nix

```
@@ -0,0 +1,26 @@
+{lib, gccStdenv, fetchFromGitHub }:
```

Suggestion: this would make the code more diff friendly

```suggestion
{ lib
, gccStdenv
, fetchFromGitHub
}:
```